Apps/frontend/src/events.js

```diff
@@ -1,2 +1,2 @@
 export const EVENT_LOADED_CHANNEL_CONFIG =
-  "event_loaded_channel_config_from_rest";
+  "event_loaded_channel_config_from_rest";
```

Comment:
What did even change?

Reply:
Nothing, maybe linting removed whitespaces or so

Reply:
`npm run format` causes for me that the whitespace is inserted again. Try running it again on your end.